nodejs/week1/src/backend/routes/meal.js

```diff
@@ -0,0 +1,15 @@
+const mealsArray = require("../data/meals.json");
+const reviewsArray = require("../data/reviews.json");
+
+const getMeal = function(request, response) {
+  const meal = mealsArray.map(mealItem => {
+    mealItem.reviews = reviewsArray.filter(
+      reviewItem => mealItem.id === reviewItem.mealId
+    );
+    return mealItem;
+  });
+  const random = Math.floor(Math.random() * mealsArray.length);
+  response.json(meal[random]);
```

Comment:
try to always state what you want to return from the function as you did on line 9
`return response.json(meal[random]);`